Update ocp-4-19-release-notes.adoc

amandverma opened the pull request:
CM sent on the 16 Sep. Will merge PR on the 17 or 18 Sep.


Version(s):
4.19

Issue:
[OSDOCS-16108](https://issues.redhat.com/browse/OSDOCS-16108)

Link to docs preview:
[Insights Runtime Extractor (Technology Preview)](https://98716--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-19-release-notes.html#ocp-release-notes-insights-operator-runtime-extractor_release-notes)

QE review:
- [x] QE has approved this change.

amandverma committed:
Update ocp-4-19-release-notes.adoc

Update release_notes/ocp-4-19-release-notes.adoc

improving as per suggestion

Co-Authored-By: Darragh Fitzmaurice <57954076+dfitzmau@users.noreply.github.com>

amandverma commented:
/label peer-review-needed

dfitzmau commented on release_notes/ocp-4-19-release-notes.adoc:
```diff
@@ -518,11 +518,10 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
-==== Insights Runtime Extractor is generally available
+==== Insights Runtime Extractor (Technology Preview)
+
+In this release, the Insights Operator introduces the workload data collection _Insights Runtime Extractor_ feature to help Red{nbsp}Hat better understand the workload of your containers. Available as a Technology Preview, the Insights Runtime Extractor feature gathers runtime workload data and sends it to Red{nbsp}Hat. Red{nbsp}Hat uses the collected runtime workload data to gain insights that c ...
```
```suggestion
In this release, the Insights Operator introduces the workload data collection _Insights Runtime Extractor_ feature. Available as a Technology Preview, the Insights Runtime Extractor feature gathers runtime workload data and sends it to Red{nbsp}Hat. Red{nbsp}Hat uses the collected runtime workload data to gain insights that you can then use to make investment decisions that helps improve how you use your {product-title} containers. For more information, see xref:../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling[Enabling features using feature gates].
```

amandverma committed:
Update ocp-4-19-release-notes.adoc

Removed "Insights Runtime Extractor is generally available" section as created new PR for adding Technology preview table for "Insights Runtime Extractor" under "Technology Preview features status"
